databricks_enums.py

```diff
@@ -0,0 +1,42 @@
+# File: databricks_enums.py
+#
+# Copyright (c) 2023 Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+#
+
+from enum import Enum
+
+
+class DatabricksEndpoint(Enum):
+    LIST_ALERTS = ('GET', '/preview/sql/alerts')
+    LIST_CLUSTERS = ('GET', '/clusters/list')
+    CREATE_ALERT = ('POST', '/preview/sql/alerts')
+    DELETE_ALERT = ('DELETE', '/preview/sql/alerts/{alert_id}')
+    PERFORM_QUERY = ('POST', '/sql/statements')
+    LIST_WAREHOUSES = ('GET', '/sql/warehouses')
+    JOB_RUN = ('GET', '/jobs/runs/get?run_id={run_id}')
+    JOB_RUN_OUTPUT = ('GET', '/jobs/runs/get-output?run_id={run_id}')
```

Comment:
you can specify query parameters by passing them to the `data` parameter in `perform_query` call instead of hardcoding them into the endpoint. they already have logic to automatically convert it to query parameters for GET requests in their SDK: https://github.com/databricks/databricks-cli/blob/112c8870b58f46360b8e0055dc5bd09fdc17ecfc/databricks_cli/sdk/api_client.py#L142